README.md

```diff
@@ -3,7 +3,8 @@
 `ada-url.com` requires Node.js 20 with [pnpm](https://pnpm.io) to install dependencies.
 
 - To install `pnpm` follow the steps in https://pnpm.io/installation.
-- To build locally, run `pnpm install` in the root repository and navigate to `http://localhost:3000`
+- To build locally, run `pnpm install` in the root repository 
+- To run the website locally, run `turbo dev` in the root repository
```

Comment:
This command will not work since it assumes the parent machine has a global turbo cli. Can you move turbo dev to run it through `pnpm run dev`

Reply:
And can you add localhost:3000 comment back?

Reply:
> And can you add localhost:3000 comment back?

Sure

> This command will not work since it assumes the parent machine has a global turbo cli. Can you move turbo dev to run it through pnpm run dev

I'm searching a better solution

Reply:
Actually I think we should just add `turbo` installation step and remove the npm requirement to run it, since `npm` takes a time to initialize as well.